### PR TITLE
Set up binder for the main branch

### PR DESCRIPTION
**Pull request by melissawm**
This brings the binder configuration (and usage guide) to the main branch of this repo.

### CODE REVIEW

**Comment by melissawm** [comment]
cc @guiwitz - I'd appreciate your review! Thanks 😄

File: napari-workshops/docs/launching_binder.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+To open a notebook from this Jupyter interface, right click the notebook name in the file navigation panel from the Jupyter interface, and click "Open with -> Notebook".
+
+![Right click on "intro_bioimage_visualization.md" file, and select "Open with -> Notebook"](./images/open_with_notebook.png)
+
+You will note that these notebooks contain a code cell which needs to be run to enable the Binder interface. 
+
+![First code cell for all notebooks, required to run the notebooks on Binder](./images/prepare_binder.png)
+
+Note that every napari operation will be shown in the desktop tab of your browser, and will only be able in the notebook interface after the `nbscreenshot` function is called. After running any cell which opens the napari viewer from the Jupyter notebook, you should now see the napari GUI window in the Desktop tab on your browser.
```
Comment (guiwitz) [inline napari-workshops/docs/launching_binder.md:35]: You wrote "and will only be able in the notebook". I'm not sure if you meant visible or enabled ?

File: .github/workflows/pages.yml [hunk 2]
```diff
@@ -30,11 +30,14 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
```
Comment (guiwitz) [inline .github/workflows/pages.yml:33]: I forgot to add a name here maybe this should be:
```
- name: Setup Python
  uses: actions/setup-python@v4
  with:
    python-version: '3.9'
```

**Review by guiwitz — APPROVED** [review]
Looks good to me! I just added two comments, one for (what I think is) a typo, the other for a minor cosmetic change to the ```pages.yml```.

**Review by chili-chiu — APPROVED** [review]
amazing! binder working well and the instruction is easy to follow. Thanks @melissawm and @guiwitz

File: napari-workshops/docs/launching_binder.md (hunk 1 above)
Comment (melissawm) [reply napari-workshops/docs/launching_binder.md:35]: Yes sorry :D

**Comment by melissawm** [comment]
Fixed! Thanks both!

**Pull request merged**